### DISCUSSION TITLE
Change the prompt of alternative media description field to be more inclusive

### DISCUSSION THREAD

**Discussion by jakobrosin** [NickColley/semaphore#33] [New features and ideas]
Currently, the field prompt says “Describe for visually impaired people.”
 Since media descriptions can be applied for videos and audio as well, the description for a particular media can be useful for different users.
Potential prompts to discuss:

- Describe the contents of uploaded media
- Describe the visuals, sounds or spoken word of the media
- Describe content for people with disabilities

**Comment by NickColley** [comment]
I think it's important that people understand who it is benefiting the most but agree it could be better so lets improve this for sure.
